docs/details/dashboards/dashboard-inventory.md

```diff
@@ -13,25 +13,40 @@ Inventory objects form a hierarchy with Node at the top, then Service and Agents
+| **Column Name**| **Description**|
+|--------------|--------------------------------|
+| Service name|The name or identifier associated with the service being monitored.|                                        
+| Node name | Name or identifier associated with a specific node.| 
+| Monitoring status| The **Monitoring** column summarizes the status of all the Agents assigned to the service.             | 
+|  Address         | The IP address or DNS where the service is currently running. |
+|  Port         | The port number on which the service is running. |
+|  QAN         | You can check Query Analytics information and the Service Overview Dashboard for each service by clicking on the <image src="../../_images/dots-three-vertical.ico" width="15px" aria-label="triple dots"/> icon in the **Options** column.|
+|  Additional information | You can also check additional information about the service, by clicking on the <image src="../../_images/arrow-downward.ico" width="15px" aria-label="downward arrow"/> icon. This expands the service entry to show reference information like service labels and IDs. |
 
-**Node Name** - Name or identifier associated with a specific node. 
 
-**Monitoring status** - The **Monitoring** column summarizes the status of all the Agents assigned to the service.
+![!image](../../_images/PMM_Inventory_Service_Selection.png)
 
-**Address** - The IP address or DNS where the service is currently running.
+#### Attributes
 
-**Port** - The port number on which the service is running.
+These are some of the atributes for a service:
 
-You can check Query Analytics information and the Service Overview Dashboard for each service by clicking on the <image src="../../_images/dots-three-vertical.ico" width="15px" aria-label="triple dots"/> icon in the **Options** column.
+- Each instance of a service gets a `service_type` attribute so one can clearly tell what type of database it is, for instance: `mysql`, `postgresql`, `mongodb`, etc. 
 
-From here you can also check additional information about the service, by clicking on the <image src="../../_images/arrow-downward.ico" width="15px" aria-label="downward arrow"/> icon. This expands the service entry to show reference information like service labels and IDs.
+- Every service is related to a certain node via its `node_id` attribute. This feature allows to support multiple instances on a single node, with different service names, e.g. `mysql1-3306`, and `mysql1-3307`.
 
-![!image](../../_images/PMM_Inventory_Service_Selection.png)
+- Starting with PMM 2.41.0, each instance of a service gets a `version` attribute to the response of the endpoint that provides a list of services being monitored by PMM. This makes it easy to visualize the database server version.
+
+    However, following are the imitations:
+    
+    - The version is not captured for the internal PostgreSQL database (currently version 14.9).
```

Comment:
```suggestion
    - The version is not captured for the internal PostgreSQL database.
```

Reply:
I'd remove the version from here so we don't have to remember to update it later on.